Gemfile

```diff
@@ -115,3 +115,7 @@ group :test do
   gem 'apivore', require: false
   gem 'hashie', '~> 3.4.6', require: false # https://github.com/westfieldlabs/apivore/issues/114
 end
+
+gem "dotenv-rails", "~> 2.6"
+
+gem "pg", "~> 0.19.0"
```

Comment:
ah - maybe you don't want these in there.  i actually run foodsoft on postgres.  and use dotenv.  but these probably don't belong in this PR